examples/otel_tracing.rs

```diff
@@ -0,0 +1,130 @@
+use anyhow::Result;
+use opentelemetry::global::shutdown_tracer_provider;
+use scylla::macros::FromRow;
+use scylla::transport::session::{IntoTypedRows, Session};
+use scylla::SessionBuilder;
+use std::env;
+use tracing::instrument::WithSubscriber;
+
+use tracing::{error, trace_span, Instrument};
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::Registry;
+
+async fn example() -> Result<()> {
+    let uri = env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session: Session = SessionBuilder::new().known_node(uri).build().await?;
+
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
```

Comment:
It would mean that much more code must be adjusted, along with other examples and tests.
Refs: 
- [session_test](https://github.com/scylladb/scylla-rust-driver/blob/main/scylla/src/transport/session_test.rs)
- [basic example](https://github.com/scylladb/scylla-rust-driver/blob/main/examples/basic.rs)

Reply:
Indeed - https://github.com/scylladb/scylladb/issues/8638 ...